spot-client/src/spot-tv/ui/components/meeting-frame/meeting-frame.js

```diff
@@ -142,9 +145,12 @@ export default class MeetingFrame extends React.Component {
      */
     render() {
         return (
-            <div
-                className = 'meeting-frame'
-                ref = { this._setMeetingContainerRef } />
+            <div className = 'meeting-frame'>
```

Comment:
I don't like this frame duplication, but I don't know any better. Is it possible to return an array here ?

Reply:
I see it can be replaced with <> to use React fragments, but I'll wait with changing that to hear from you if it's ok to do it in meeting-frame at all

Reply:
Yeah, it's okay. We run fast and loose here, but more importantly using a fragment is idiomatic anyway.